src/pickPackage.ts

```diff
@@ -58,12 +61,15 @@ export default async (
   },
 ): Promise<{meta: PackageMeta, pickedPackage: PackageInRegistry | null}> => {
   opts = opts || {}
+  ctx.cacheTtl = ctx.cacheTtl || DEFAULT_CACHE_TTL
```

Comment:
the default value can be assigned in `src/index.ts`, in that case it can be a non-nullable value